Formula/ask-cli.rb

```diff
@@ -0,0 +1,19 @@
+require "language/node"
+
+class AskCli < Formula
+  desc "CLI tool for Alexa Skill Kit"
+  homepage "https://www.npmjs.com/package/ask-cli"
+  url "https://registry.npmjs.org/ask-cli/-/ask-cli-1.1.2.tgz"
+  sha256 "6f3f4d4ad440001c2ffed0885aafb9c0ce1df73da9213465631e4ee9df677c1d"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system bin/"ask", "--help"
```

Comment:
Can we have a test that does something more? Like tries to connect without credentials and match the error message.

Reply:
@fxcoudert sure, I add a better test for this formula